src/test.cpp

```diff
@@ -132,4 +132,43 @@ void Test::settingsManager()
              qPrintable(QString("Distance scalar in meters is wrong: %1").arg(scalar)));
 }
 
+void Test::waterfallGradient()
+{
+    QVector<QColor> colorList = {Qt::black, Qt::white};
+    // Check name and validation
+    auto testName = QString("Test");
```

Comment:
`"Waterfall gradient test"`?

Reply:
It's only a string to create a valid waterfall, the only thing that is necessary here is not to be empty.